tmk_core/common/chibios/pin_defs.h

```diff
@@ -0,0 +1,242 @@
+/* Copyright 2021 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+// Defines mapping for Proton C replacement
+#ifdef CONVERT_TO_PROTON_C
```

Comment:
Would it be possible to add the non-pro micro pins as defines here, as well?  Such as the pins on the break away, and "inside"?

Reply:
Possible yes, though i would rather keep the scope down for the individual PRs. Ive got a 2nd iteration on top of this onethat moves the CONVERT_TO_PROTON_C out, so more "converters" could be added at a later date.